test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SigningUtilityTests.cs

```diff
@@ -109,22 +109,21 @@ public void Verify_WhenChainBuildingFails_Throws()
                 Assert.Equal("Certificate chain validation failed.", exception.Message);
 
                 Assert.Equal(1, logger.Errors);
-                Assert.Equal(1, logger.Warnings);
 
-                if (RuntimeEnvironmentHelper.IsWindows)
+                if (!RuntimeEnvironmentHelper.IsMacOSX && RuntimeEnvironmentHelper.IsLinux)
```

Comment:
Are both of these conditions necessary?  Why is the following inadequate?
```C#
if (RuntimeEnvironmentHelper.IsLinux)
```
In the past we've had a [problem](https://github.com/NuGet/Home/issues/4648) where `IsLinux` returned `true` on macOS.  I don't know if that's still true.

Reply:
Thanks! IsLinux returns false on macOS. Changed accordingly.